config/default.json

```
@@ -0,0 +1,13 @@
+{
+  "amqp": {
+    "hostname": "localhost",
+    "port": 5672,
+    "username": "knot",
+    "password": "knot",
+    "token": ""
+  },
```

We no longer have `token` config for `amqp`. Instead, its passed as a parameter now.

```suggestion
  "amqp": {
    "hostname": "localhost",
    "port": 5672,
    "username": "knot",
    "password": "knot"
  },
```

---

Done!